src/main/scala/Domain/Door.scala

```diff
@@ -56,7 +56,7 @@ object Door {
       }
 
   val aggregateLogic: List[Flow[Unit]] = List(
-    handler { case Register(id) => emitEvent(Registered(id)) } >> waitFor { case Registered(_) => () },
+    handler { handler[Register, Registered] } >> waitFor { case Registered(_) => () },
```

Comment:
I agree! What about?
- `pass`: you allowed to pass on
- `promote`: promoting a command to a event

`direct` is also good if you keep the validation part in mind :)